scripts/release.ts

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env zx
 
 import { versionBump } from 'bumpp'
-import glob from 'fast-glob'
-import { $ } from 'zx'
+import { $, glob } from 'zx'
 
 try {
-  const packages = await glob(['package.json', 'server/package.json', 'package/package.json'])
+  const packages = await glob(['package.json', '*/package.json'])
```

Comment:
Does this exclude `node_modules` by default?

Reply:
No. But the pattern isnt deep (**/package.json), so excluding `node_modules` is not required here. Anyway, I can revert this change if it seems unreliable.